zio-http/src/test/scala/zhttp/internal/DynamicServer.scala

```diff
@@ -1,16 +1,16 @@
 package zhttp.internal
 
 import zhttp.http._
+import zio._
 import zio.blocking.Blocking
 import zio.console.Console
-import zio.{Ref, UIO, ZIO, ZLayer}
 
 import java.util.UUID
 
-object AppCollection {
+object DynamicServer {
```

Comment:
```suggestion
object ReusableServer {
```

An alternative name to consider (Optional)

Reply:
I think DynamicServer is better. 😅